caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/ArtifactHarvester.java

```diff
@@ -203,12 +210,14 @@ public Integer run() throws Exception {
 
                                 } catch (Throwable t) {
                                     success = false;
-                                    message = "Failed to determine if artifact " + artifact.getURI() + " exists: " + t.getMessage();
+                                    message = "Failed to determine if artifact " + artifact.getURI() + " exists: " +
+                                        t.getMessage();
                                     log.error(message, t);
                                     if (!dryrun) {
                                         log.debug("Adding artifact to skip table: " + artifact.getURI());
                                         // set the message to be an empty string
-                                        HarvestSkipURI skip = new HarvestSkipURI(source, STATE_CLASS, artifact.getURI(), "");
+                                        HarvestSkipURI skip = new HarvestSkipURI(source, STATE_CLASS, artifact.getURI
+                                            (), "");
```

Comment:
Quite a few line break changes here, which is okay, but I don't know about these floating parentheses...